python/lib/metrics/__init__.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import typing
+
+from argparse import ArgumentParser
+
+
+class Arguments(object):
```

Comment:
This is for sure not critical, (I am assuming this is code re-use) we can get rid of `object`.

Reply:
Also, the shebang since this will not be run from the command line directly.

Reply:
Ive noticed that several places, but just have not commented on it. I think it makes the most sense to open an issue to clean unnecessary shebangs in the future.